src/components/shared/ListView/ListView.tsx

```diff
@@ -16,11 +16,9 @@ export interface IDefaultResource {
   LastName?: string
 }
 
-export interface ListViewTableOptions<T>
-  extends Omit<IDataTable<T>, "data" | "selected" | "handleSelectionChange" | "rowActions" | "onSortChange"> {}
+export interface ListViewTableOptions<T> extends Pick<IDataTable<T>, "columns" | "responsive" | "hideColumns"> {}
```

Comment:
I used Omit here because I was "omitting" the properties I do not want the developer to change through ListView.tableOptions. By using pick here, you are limiting the tableOptions severely and I don't think that will fly for our usage of ListView in BUT. To keep our stuff as compatible as possible, please undo this change.